package.json

```diff
@@ -9,7 +9,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@polkadot/api": "6.3.1",
+    "@polkadot/api": "^10.9.1",
```

Comment:
Have we tested this post change? Seems a major dependency update.

Reply:
This matches dot SDK, since code is identical it shouldn't be a problem. I also tested `bond`, `bondExtra`, `chill`, `addProxy` and `unbond`